docs/UserGuide.md

```diff
@@ -185,8 +194,17 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 Examples:
 * `find John` returns `john` and `John Doe`
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+
+**Image Example**
+
+Command: ` find John Stanley `
+
+Before the find command ran:
+<br><img src="images/FindCommandBefore.png" width = "300">
+
+After the find command ran:
+<br><img src="images/FindCommandAfter.png" width = "300">
```

Comment:
nice of you to write out the actual command, include before and after scenario for the find command!